### ISSUE TITLE
Configure logging for tornado.access

### ISSUE THREAD

**Issue by thomir** [mitmproxy/mitmproxy#462]
While using mitmproxy to capture traffic, I sometimes see the message:

```No loggers configured for tornado.access```

This is printed when the python logging system receives a log message for a logger that hasn't been configured. This warning message is written over the traffic I'm trying to view, and is rather annoying. It would be nice if mitmproxy at least configured all loggers to log to a null hander, so this message didn't overwrite the content being presented.

**Comment by mhils** [comment]
Hi there,

thanks for the report! Unfortunately, @cortesi and I are fairly busy right now. We'll certainly get back to your issue, but it may take a bit - sorry for that.

Cheers,
Max

**Comment by mhils** [comment]
Hi @thomir,

thanks for the report. How can I reproduce this bug?

Cheers,
Max

**Comment by thomir** [comment]
hmmm, it's not happening for me any more - which is odd. I'll update the bug if I see this again....

**Comment by ibash** [comment]
I got this when attempting to download `/cert/pem` on my iphone. The logger for the onboarding app was not configured. After adding a basic logger the error I was getting was:
```
WARNING:tornado.access:405 HEAD /cert/pem (192.168.88.186) 0.53ms
```

I added a head route to `/cert/pem` that did nothing, then when I tried to reinstall the cert I got a "could not determine file size" and was able to download it anyway.

Leading up to this I previously had the profile installed and then removed it, as well as uninstalled and reinstalled mitmproxy a few times.
